lib.bash

```diff
@@ -178,13 +178,20 @@ pool_state_color() {
   esac
 }
 
+# ZFS in FreeBSD 13 has changed slightly; `zpool status` does not
+# output a `scan` line unless a scrub has been requested.
 scrub_info() {
   local pool=$1
-
-  zpool status "$pool" \
-    | grep -A 2 '^  scan: ' \
-    | grep -E '(^        )|(scan: )' \
-    | sed -E 's/^.{8}//'
+  local info=$( zpool status "$pool" \
```

Comment:
```suggestion
  local info
  info=$( zpool status "$pool" \
```

when using the `set -e` flag, it will cause a line that has a non-zero status to exit/return immediately from a function, however the `local` keyword will mask this and the line will return a zero status because the `local` declaration succeeded.

to work around this, when using command substitution (ie: `$( ... )`), declare the variable, then assign it as a separate statement.

Reply:
Fixed in latest push.